packages/astro/src/react/hooks.ts

```diff
@@ -121,8 +121,6 @@ type UseAuth = () => UseAuthReturn;
  * For projects using a server, you can have immediate access to this data during SSR.
  *
  * @example
- * A simple example:
```

Comment:
The "A simple example" where removed because they'd otherwise would show up in the example code. We can already give context that it's an example without that sentence